keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ResourceController.kt

```diff
@@ -72,6 +74,22 @@ class ResourceController(
     return resourceRepository.getStatus(id)
   }
 
```

Comment:
@erikmunson these methods on the resource controller are for pausing/resuming a specific resource